interface/frontend/src/components/open-image/ImageSeries.vue

```diff
@@ -80,6 +87,7 @@
   import { EventBus } from '../../main.js'
   import TreeView from './TreeView'
   import OpenDicom from './OpenDICOM'
+  const dirname = require('path-dirname')
```

Comment:
Hmm, seems like this module cannot be used with `import`? I would imagine `import dirname from 'path-dirname'` woudl work.

Reply:
Well, yes, it works like you thought. I was confused by this [this](https://github.com/concept-to-clinic/concept-to-clinic/pull/188#discussion_r147443976) obstacle for the `cornerstone` and `cornerstone-tools`.